jest.config.js

```diff
@@ -1,6 +1,9 @@
 module.exports = {
   collectCoverage: true,
   collectCoverageFrom: ['./src/**.ts'],
+  // Jest (i.e. istanbul) identifies every export in index.ts as a distinct
+  // function, which tanks our function coverage.
+  coveragePathIgnorePatterns: ['index.ts'],
```

Comment:
Kinda think we should still count this in our coverage report though :thinking: This is what e2e tests would cover. It's important to make sure our exports exist and are named correctly and so on. This would catch the ES interop issue we've had many times as well, where we broke exports accidentally.

This is fine for now though.